### ISSUE TITLE
Max. number of airplay1 speakers

### ISSUE THREAD

**Issue by beaune33** [ejurgensen/forked-daapd#1198]
Hello,

I have been using forked-daapd with 4 airplay 1 speakers successfully. Now I added a 5th one, and it gets instable; speakers continue switching on and off. As soon as I limit the number of speakers to 4, it gets stable again, independand of which of the speakers is switched off

I am wondering what may be the cause. My wifi is a slow one with 50MB, I know. But on the other hand this should fit for 5 speakers, shouldn't it? Therefore I would like to ask, if anyone is using 5 or more speakers on one wifi network, if there is any known issue in forked-daapd, anything I should check etc. I would be glad if anybody could give me a hint what could be a reason for this behaviour, or maybe what I could analyse.

Best regards
Beaune

**Comment by ejurgensen** [comment]
Yes, it could be bandwidth - I assume these are Airplay speakers? There the stream is genereally uncompressed, which can become heavy with many speakers.

I have actually added lossless compression which approximately halves the required bandwidth, but it is very recent and you might not have it available in the version you are running. It is also only for Airplay 2 capable devices. Which version are you running, and on which platform?

**Comment by beaune33** [comment]
You are right, they are airplay-1-speakers, in detail Philips Fidelio DS8800/DS3800 and AD7000. I am running forked daapd on version 27.2 on a Raspi 4 (buster) (kernel: LINUX 5.4.83-v71+). What would I have to do to test the lossless compression, which forked-daapd version is needed? Or doesn't that make sense because of airplay version 1 devices? Then the solution would be a faster wifi or a second wifi access point?

**Comment by ejurgensen** [comment]
I can't say for sure that the issue is bandwidth related.

One small thing you could try is to set the config option "raop_disable = true" for one of your speakers. Then restart forked-daapd. If the speaker is now gone from the speaker list then it really only has Airplay 1, and I'm not planning on adding compression for that.

**Comment by beaune33** [comment]
I did two tests:

- Checking of airplay version, as you exlained above. Indeed my speakers are airplay 1 only, as I expected. The one with raop_disable = true disappeared from the forked-daapd-list.
- I used a second wifi router and connected the 5th speaker to that one. Then all speakers are working.

It seems you are right with your assumption, that the issue is bandwith related. Its really a pity that the compression is not available for airplay1 devices, as that would most probably fix my issue. Maybe I could put something on to the wish list for christmas ;-) ?

**Comment by ejurgensen** [comment]
Closed with above commit
